ui/perfherder/CompareTable.jsx

```diff
@@ -0,0 +1,198 @@
+      }`;
+    if (type === 'text' && className) return `text-${className}`;
+    return className;
+  };
+
+  deltaTooltipText = (delta, percentage, improvement) =>
+    `Mean difference: ${displayNumber(delta)} (= ${Math.abs(
+      displayNumber(percentage),
+    )}% ${improvement ? 'better' : 'worse'})`;
+
+  render() {
+    const { data, testName, title } = this.props;
+    return (
+      <Table sz="small" className="compare-table mb-0" key={title || testName}>
+        <thead>
+          <tr
+            style={{ backgroundColor: 'lightgrey' }}
+            className="subtest-header"
+          >
+            <th className="text-left">
+              <span>{testName}</span>
+            </th>
+            <th style={{ width: '200px' }}>Base</th>
+            {/* empty for less than/greater than data */}
+            <th style={{ width: '30px' }} />
+            <th style={{ width: '140px' }}>New</th>
+            <th style={{ width: '100px' }}>Delta</th>
+            {/* empty for progress bars (magnitude of difference) */}
+            <th style={{ width: '120px' }} />
+            <th style={{ width: '100px' }}>Confidence</th>
+            <th className="text-right" style={{ width: '80px' }}>
+              # Runs
+            </th>
+          </tr>
+        </thead>
+        <tbody>
+          {data.map(results => (
+            <tr key={results.name}>
+              <th className="text-left font-weight-normal pl-1">
+                {results.name}
+                {results.links && (
+                  <span className="result-links">
+                    {results.links.map(link => (
+                      <span key={link.title}>
+                        <a href={link.href}>{` ${link.title}`}</a>
+                      </span>
+                    ))}
+                  </span>
+                )}
+              </th>
+              <TableAverage
+                value={results.originalValue}
+                stddev={results.originalStddev}
+                stddevpct={results.originalStddevPct}
+                replicates={results.originalRuns}
+              />
+              <td>
+                {results.originalValue < results.newValue && <span>&lt;</span>}
+                {results.originalValue > results.newValue && <span>&gt;</span>}
+              </td>
+              <TableAverage
+                value={results.newValue}
+                stddev={results.newStddev}
+                stddevpct={results.newStddevPct}
+                replicates={results.newRuns}
+              />
+              <td className={this.getColorClass(results, 'background')}>
+                {results.delta &&
+                Math.abs(displayNumber(results.deltaPercentage)) !== 0 ? (
+                  <SimpleTooltip
+                    textClass="detail-hint"
+                    text={
+                      <React.Fragment>
+                        {(results.isRegression || results.isImprovement) && (
+                          <Icon
```

Comment:
Using the new `@fortawesome/react-fontawesome` package, this would be something like:

```js
<FontAwesomeIcon
  icon={results.isRegression ? faExclamationTriangle : faThumbsUp}
  size="..."  // replaces `fontSize`, see https://fontawesome.com/how-to-use/on-the-web/styling/sizing-icons
  // other attributes stay the same
/>
```